test/UnitTest/System.Web.OData.Test/OData/Query/Expressions/SelectExpandBinderTest.cs

```diff
@@ -74,7 +74,7 @@ public void Bind_GeneratedExpression_ContainsExpandedObject()
             Assert.True(enumerator.MoveNext());
             var partialCustomer = Assert.IsAssignableFrom<SelectExpandWrapper<Customer>>(enumerator.Current);
             Assert.False(enumerator.MoveNext());
-            Assert.Null(partialCustomer.Instance);
+            Assert.NotNull(partialCustomer.Instance);
```

Comment:
Can you add some additional validation to ensure Instance is set to the correct object?